src/edu/ucsb/cs56/projects/games/fish_animation/FishAnimationEnvironment.java

```diff
@@ -189,6 +189,15 @@ private Bubbles createBubbles(int x, int y, int diameter) {
 	 * creates Jellyfish based on the selected difficulty.
 	 */
 	public FishAnimationEnvironment(boolean character_type, int difficulty, boolean l) {
+		
+		if (character_type){
+			maxHealth = 50;
+			health = 50;
+		}
+		else{
+			maxHealth = 100;
+			health = 100;}
```

Comment:
Pull this curly bracket down to the next line.

Reply:
fixed